Lombiq.Tests.UI/Extensions/IWebApplicationInstanceExtensions.cs

```diff
@@ -50,7 +50,10 @@ public static class IWebApplicationInstanceExtensions
             httpContextAccessor.HttpContext = shellScope.ShellContext.CreateHttpContext();
             var httpContext = httpContextAccessor.HttpContext;
 
+            // Seems to be a false positive warning. IDE thinks that left hand side can't be null.
+#pragma warning disable S2583 // Conditionally executed code should be reachable
             httpContext.Request.PathBase = "/" + shellHost.GetSettings(tenant).RequestUrlPrefix ?? string.Empty;
+#pragma warning restore S2583 // Conditionally executed code should be reachable
```

Comment:
It's not a false positive, but a precedence issue. It turns out the `+` is evaluated before the `??` while I think the intention for this code was to do `"/" + (shellHost.GetSettings(tenant).RequestUrlPrefix ?? string.Empty)`.
![image](https://github.com/Lombiq/UI-Testing-Toolbox/assets/4007293/a10e76ad-bc5e-4782-bd32-56ab885e8a5a)

Also it turns out that the `?? string.Empty` is pointless, because adding null to a string is already the same as adding an empty string to a string:
![image](https://github.com/Lombiq/UI-Testing-Toolbox/assets/4007293/938445e9-790d-4f2a-8066-b24a542bdb6c)